scripts/configure-ironic.sh

```diff
@@ -86,6 +86,9 @@ mkdir -p /shared/ironic_prometheus_exporter
 
 configure_json_rpc_auth
 
+ENABLE_FIPS_IPA=$(cat /proc/sys/crypto/fips_enabled)
```

Comment:
Is this file guaranteed to exist on Centos? It does not exist on my Ubuntu machine, which would result in empty variable, instead of 0 or 1.

Reply:
this should be guarantee existing in FIPS enabled kernels, including ubuntu, since at least 4 years
I don't recall exactly the kernel version, but I'm quite sure the file exists since at least RHEL/CENTOS 7.6 and Ubuntu 16.0.4

Reply:
found it, a bit older than 4 years :)
https://www.kernelconfig.io/config_crypto_fips

Reply:
It still isn't present on Ubuntu 22.04 by default tho. Obviously my desktop kernel isn't FIPS enabled.

Whats the behavior of the flag if it isn't 0 or 1 but empty? `simple_strtol` used by fips.c isn't well documented.

Reply:
probably only very latest kernels are all compiled with fips support by default
I guess I can add a test for the file, just to be on the safe side